test/integration/ynEIGEN/EigenStrategyManager.t.sol

```diff
@@ -184,6 +303,25 @@ contract EigenStrategyManagerTest is ynEigenIntegrationBaseTest {
         }
 
     }
+    
+    
+    function testStakeNodesAndSlash() public {
```

Comment:
Also add tests for:
- strategiesBalance is not updated if not synchronized
- strategiesBalance is decreased by 50% when half the deposits are queued for withdrawal, slashed and sync.